src/imageio.cs

```diff
@@ -2356,4 +2368,28 @@ interface CGImageAuxiliaryDataInfo {
 		NSData Data { get; set; }
 		NSDictionary DataDescription { get; set; }
 	}
+
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	[Static]
+	[Internal]
+	interface CGImageAnimationOptionsKeys {
+		[Field ("kCGImageAnimationDelayTime")]
+		NSString DelayTimeKey { get; }
+
+		[Field ("kCGImageAnimationLoopCount")]
+		NSString LoopCountKey { get; }
+
+		[Field ("kCGImageAnimationStartIndex")]
+		NSString StartIndexKey { get; }
+	}
+
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	[StrongDictionary ("CGImageAnimationOptionsKeys")]
+	interface CGImageAnimationOptions {
+		double DelayTime { get; set; }
+
+		NSNumber LoopCount { get; set; }
+
+		NSNumber StartIndex { get; set; }
```

Comment:
same for `kCGImageAnimationLoopCount` it is used as follows

```objective-c

@property (nonatomic) NSUInteger beginningFrameIndex;
...
options[(NSString *) kCGImageAnimationStartIndex] = @(self.beginningFrameIndex);
```

```suggestion
		nuint StartIndex { get; set; }
```